### PR TITLE
Use Spring placeholders to access datadir properties

### PR DESCRIPTION
**Pull request by severo**
instead of the GeorchestraConfiguration class.

### CODE REVIEW

**Comment by severo** [comment]
Note that I used the same pattern as in #2494 to get access to the language property from inside a JSP file

**Comment by severo** [comment]
See #2496

File: header/src/main/webapp/WEB-INF/ws-servlet.xml [hunk 1]
```diff
@@ -15,10 +15,20 @@
...
+            </map>
+        </property>
     </bean>
 
     <bean id="georchestraConfiguration" class="org.georchestra.commons.configuration.GeorchestraConfiguration">
         <constructor-arg value="header" />
     </bean>
 
+    <context:property-placeholder
+        location="file:${georchestra.datadir}/default.properties, file:${georchestra.datadir}/mapfishapp/mapfishapp.properties"
```
Comment (fvanderbiest) [inline header/src/main/webapp/WEB-INF/ws-servlet.xml]: you probably mean header.properties ;-)

Comment (severo) [reply header/src/main/webapp/WEB-INF/ws-servlet.xml]: AHHHHH

Comment (fvanderbiest) [reply header/src/main/webapp/WEB-INF/ws-servlet.xml]: You're allowed to `git push --force` in the branch ;-)

Comment (severo) [reply header/src/main/webapp/WEB-INF/ws-servlet.xml]: Yes. Done.

**Pull request merged**